packages/langchain/src/openai/util.ts

```diff
@@ -146,6 +146,7 @@ export function mapOutputToChatResult(
       id: completionResponse.id,
       model: completionResponse.model,
       object: completionResponse.object,
+      promptFilterResults: completionResponse.prompt_filter_results ?? [],
```

Comment:
```suggestion
      promptFilterResults: completionResponse.prompt_filter_results,
```

Thought about this, just assigning should also be enough imo.